forge-gui/res/cardsfolder/v/volcano_hellion.txt

```diff
@@ -5,7 +5,7 @@ PT:6/5
 Text:CARDNAME has echo X, where X is your life total.
 K:Echo:X
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDmg | TriggerDescription$ When CARDNAME enters the battlefield, it deals an amount of damage of your choice to you and target creature. The damage can't be prevented.
-SVar:TrigDmg:DB$ ChooseNumber | ListTitle$ an amount of damage to you and target creature | ChooseAnyNumber$ True | SubAbility$ DBDmg | AILogic$ Min
+SVar:TrigDmg:DB$ ChooseNumber | ListTitle$ Choose amount of damage to be dealt | ChooseAnyNumber$ True | SubAbility$ DBDmg | AILogic$ Min
```

Comment:
these changes need adjustment:
![image](https://github.com/Card-Forge/forge/assets/8506892/11eebcbb-9fd4-4e23-a276-ec52af256660)

Reply:
Oops. Hopefully fixed to satisfaction. 
I took my lead from the pattern established `ChoiceTitle$`. I do hope either that one or `RevealTitle$` aren't amiss.

Reply:
thanks
I'd have to check ingame but I'm gonna slowly start merging the other PR
feel free to update them before or in follow-up

Reply:
Had a look to how things were on my side, and no surprises for those as far as I could tell.